tests/python/mints13/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
 include(TestingMacros)
 
-add_regression_test(python-mints13 "psi;mints;quicktests;python")
+add_regression_test(python-mints13 "psi;mints;quicktests;python;cart")
```

Comment:
What is `cart`?

Reply:
Those are the test cases whose Mols use no zmat spec or variables or efp or pubchem, just plain Cartesian. It was tedious enough to find and add them that I left them in for future.